integ/components/deadline/deadline_04_workerFleetHttps/lib/workerFleetHttps-testing-tier.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path';
+import { Port } from '@aws-cdk/aws-ec2';
+import { Construct } from '@aws-cdk/core';
+import { IWorkerFleet } from 'aws-rfdk/deadline';
+import { TestingTier, TestingTierProps } from '../../../../lib/testing-tier';
+import { WorkerStruct } from '../../../../lib/worker-struct';
+
+/**
+ * Interface for WorkerFleetTestingTier properties
+ */
+export interface WorkerFleetTestingTierProps extends TestingTierProps {
+  /**
+   * Array of WorkerStructs representing different test cases
+   */
+  readonly structs: Array<WorkerStruct>;
+}
+
+/**
+ * Testing Tier for the Deadline WorkerFleet HTTPS integration test
+ *
+ * Creates a test bastion and configures it to connect to one or more Deadline WorkerInstanceFleet constructs for testing.
+ *
+ * Resources Deployed
+ * ------------------------
+ * - A BastionLinuxHost instance
+ *
+ * Security Considerations
+ * ------------------------
+ * - The bastion instance created by this test is configured to access farm resources on their default ports
+ *   Test scripts stored on the bastion are used to submit Deadline jobs to farm workers and request information about the workers.
+ *   At execution the tests retrieve the value of secrets for the authentication cert.
+ */
+export class WorkerFleetTestingTier extends TestingTier {
+  constructor(scope: Construct, id: string, props: WorkerFleetTestingTierProps) {
+    super(scope, id, props);
+
+    const structs = props.structs;
+    structs.forEach( workerStruct => {
+
+      const testSuiteId = 'WFS' + (structs.indexOf(workerStruct) + 1).toString();
```

Comment:
This file is almost an exact copy of `integ/components/deadline/deadline_03_workerFleetHttp/lib/workerFleetHttp-testing-tier.ts`. The only functional change is this `testSuiteId`. Can we make this a shared file and take that as a prop?